src/style.css

```diff
@@ -1 +1,52 @@
-/* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+/* .container {
+  width: 96px;
+
+  height: 96px;
+  display: flex;
+  background-color: #fff;
+  flex-direction: column;
+  justify-content: space-between;
+} */
+
+.stars {
+  display: flex;
+  margin: 0;
+  list-style: none;
+  padding: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
You can use short version - https://developer.mozilla.org/ru/docs/Web/CSS/background